src/modules/familyMember/application/familyMember.service.ts

```diff
@@ -62,29 +60,80 @@ export class FamilyMemberService {
         if (!existingFamilyMember) {
             throw new NotFoundException(`Family member with ID ${id} not found.`);
         }
+
         if (dto.registrationNumber) {
+            if (!UpdateFamilyMemberDTO.validateCPF(dto.registrationNumber)) {
+                throw new BadRequestException("Invalid CPF");
+            }
             const regOwner = await this.familyMemberRepository.findByRegistrationNumber(dto.registrationNumber);
             if (regOwner && regOwner.getId() !== id){
-                throw new ConflictException("CPF '${dto.registrationNumber}' already in use!");
+                throw new ConflictException(`CPF '${dto.registrationNumber}' is already in use.`);
             }
         }
         if (dto.email) {
             const emailOwner = await this.familyMemberRepository.findByEmail(dto.email);
             if (emailOwner && emailOwner.getId() !== id) {
-                throw new ConflictException(`Email '${dto.email}' already in use!`);
+                throw new ConflictException(`Email '${dto.email}' is already in use.`);
+            }
+        }
+
+        if (dto.dateOfBirth) {
+            if (dto.dateOfBirth > new Date()) {
+                throw new BadRequestException('Date of birth cannot be in the future.');
             }
         }
+
+        Object.keys(dto).forEach(key => {
+            const setterName = `set${key.charAt(0).toUpperCase() + key.slice(1)}`;
+            if (typeof existingFamilyMember[setterName] === 'function') {
+                existingFamilyMember[setterName](dto[key]);
+            }
+        });
```

Comment:
```suggestion
        Object.keys(dto).forEach(key => {
            if(dto[key]){
                const setterName = `set${key.charAt(0).toUpperCase() + key.slice(1)}`;
                if (typeof existingFamilyMember[setterName] === 'function') {
                    existingFamilyMember[setterName](dto[key]);
                }
            }
        });
```
Ao enviar um valor como:
```
{
    "fullName": "Joana Silva"
}
```
A aplicação estava retornando um erro 500.